### ISSUE TITLE
`focus-visible` polyfill breaking some button focus styles in Firefox

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-frontend#766]
## Description

Buttons are getting double focus rings. I think this is due to the focus-visible polyfill. If it is, then a simple focus could be to only load that polyfill for Safari, it's the only browser that needs it anyway.

<img width="460" alt="Captura de Tela 2022-02-01 às 07 55 34" src="https://user-images.githubusercontent.com/24264157/151972392-db23b0fb-8a05-4aab-97ff-084a993bda3b.png">

## Reproduction

1. Run `main` and visit a search page in a small viewport width
2. Open the content type modal
3. Focus the content types with the keyboard
4. Note the double focus outlines

## Expectation

There should be no double focus outlines.

## Environment

 - Device: laptop
 - OS: macOS 11.6.2
 - Browser: Firefox
 - Version: 96.0.3

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.
